js-exercises/fibonacci-iter/fibonacciIter.js

```diff
@@ -0,0 +1,15 @@
+var fibonacciIter = {
+  [Symbol.iterator]() {
+    let n1 = 1, n2 = 1, x;
```

Comment:
you could have used a good nomenclature for the following. 
`let firstNumber = 1, secondNumber = 2, currentNumber;`
That would have improved the readability.